package.json

```diff
@@ -33,6 +32,7 @@
     "jstransform": "latest",
     "lodash.pickby": "^4.6.0",
     "object.assign": "^4.0.4",
+    "regenerator-runtime": "^0.13.2",
```

Comment:
Why is this needed st the top level?

Reply:
For replacement of deprecated and obsolete `babel-polyfill`.

Reply:
what about `@babel/polyfill`?

Reply:
either way everything is labeled as “with core-is”, not “with regenerator-runtime”. Should we remov this and update the test results?

Reply:
`@babel/polyfill` is dead. `regenerator` transform is a part of `babel`, I don't think that any additional labels required.